content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -63,7 +63,7 @@ An _output entry_ can consist of these fields:
 
 A _dependency entry_ consists of a these possible fields:
 
-- `path`: Path to the dependency, relative to the `wdir` path (always present)
+- `path`: Path to the dependency, relative to the `.dvc` file location
```

Comment:
@efiop or perhaps @skshetry can you please confirm this?

Reply:
It is still relative to the `wdir`.

Reply:
OK... But there's no longer `wdir` in .dvc files, is there? Only in dvc.yaml

So for .dvc files it's relative to the file location and for dvc.yaml its relative to `wdir` (which defaults to the dvc.yaml file location)?

Reply:
wdir is not explicitly present, but it could still be there (same with dvc.yaml). Path resolution logic stays the same.

Reply:
OK, I see. But the .dvc or dvc.yaml location is the default wdir, right?

Reply:
@jorgeorpinel `wdir` is resolved relative to the dvcfile location, and `wdir` is `.`(curdir) by default, so yeah.

Reply:
Got it.